lib/ioh-guest

```diff
@@ -576,7 +576,7 @@ __guest_uefi() {
 	# Create tap if needed
 	__tap_setup $dataset
 	# Add tap interface
-	local tappci="$pci $(__tap_pci $dataset)"
+	local tappci="$pci$(__tap_pci $dataset)"
```

Comment:
There is no definition of local pci variable, so $pci should be removed.
local tappci="$(__tap_pci $dataset)"